user_docs/en/userGuide.t2t

```diff
@@ -2784,18 +2784,18 @@ You can browse installed incompatible add-ons using the [incompatible add-ons ta
 + Extra Tools +[ExtraTools]
 
 ++ Log Viewer ++[LogViewer]
-The log viewer, found under Tools in the NVDA menu, allows you to view the logging output that has occurred since the last session of NVDA was started.
+The log viewer, found under Tools in the NVDA menu, allows you to view the logging output that has occurred since the current session of NVDA was started.
 
 Apart from reading the content, you can also Save a copy of the log file, or refresh the viewer so that it loads new log output generated after the Log viewer was opened.
-These actions are available under the Log viewer's menu.
+These actions are available under the Log submenu in the log viewer.

```

Comment:
It's not a submenu, technically its a "toolbar menu"

```suggestion
These actions are available under the Log menu in the log viewer.
```

Reply:
@seanbudd the synthesizer how ever pronounces it as "log submenu". Should I commit your change though?

Reply:
No I suppose what is announced is easier to understand. I just figured some may be confused, and expect it as a submenu under NVDA Menu > tools > logs > ?

Reply:
More precisely, "Log" is a menu item (the unique one) in the menu bar of the log viewer's window. If you press Alt and then NVDA+tab, you will have the full object's description reported:
"Log  menu item  focused  subMenu  Alt+l"

But when navigating, the type of object (menu item) is not reported to avoid excessive verbosity.

"submenu" is not describing the type (or role) of the focused object, since this role is referred as menu item instead. "Submenu" is just here to indicate a property (or state) of this object that indicates that this object has a submenu, i.e. opens a new menu. "submenu" is short for "has submenu" and is the way the state "HASPOPUP" is reported.

If you enter this submenu, arrow down to "Refresh" (or any other item) and press NVDA+numpad8 to got to the parent object, it is referred as "Log menu". Thus these items are in a menu. A submenu is not an intrinsic type of object; it rather describes an object with respect to another one.

Last but not least: in the majority of material describing actions in menu bars and items, such sentence would use the term menu and not submenu. E.g. the "file" menu, the "edit" menu, etc.
Thus, I really think that @seanbudd's suggestion should be committed.

Reply:
I understand your point, however the user will understand what is reported by NVDA and will make reference to that piece of reporting. What you say is better suited in the developers guide in my view. The synth reports "log submenu" when pressing alt key in the log viewer. We cannot have it clearer than that, otherwise we have to change the reporting message sent to the synth ant reflect the new reporting message in the user guide.

Reply:
> I understand your point, however the user will understand what is reported by NVDA and will make reference to that piece of reporting.

Again, I have never heard people being confused when reading instructions such as: "Go to File menu and select Open".  Writing "submenu" instead of "menu" just make things more complicated in my view.

Anyway let's not lose time anymore on a single word, that's not so important. I have given details on my opinion. Put the word you want.